x/dogfood/types/expected_keepers.go

```diff
@@ -94,4 +94,5 @@ type AssetsKeeper interface {
 type AVSKeeper interface {
 	RegisterAVSWithChainID(sdk.Context, *avstypes.AVSRegisterOrDeregisterParams) (common.Address, error)
 	IsAVSByChainID(ctx sdk.Context, chainID string) (bool, common.Address)
+	AVSInfoUpdate(ctx sdk.Context, params *avstypes.AVSRegisterOrDeregisterParams) error
```

Comment:
It's better to rename this AVSInfoUpdate into UpdateAVSInfo

Reply:
fixed